### DISCUSSION TITLE
Dark Theme for the app?

### DISCUSSION THREAD

**Discussion by Proutbedaine** [MalloyDelacroix/DownloaderForReddit#259] [Ideas]
I think there's not enough app that offer this now that we're in 2021, but a dark theme for any app should be a standard no? I mean, I do admit I have no idea how hard it can possibly be to do such a thing. Let's say, make it look like GitHub for example! I think this is one of the best example of color pallet for a dark theme and how they use it too; but doing something like that for these kinds of app, is it possible? And if so how hard is it? 

Because personally I would highly appreciate it to be a thing. I really like to use this app full screen. The thing is though, I'm on a QLED 55" that can get REALLY bright if everything is white. With dark themes and any other stuff (movies, YouTube, gaming) the level of brightness is perfect and make every color pop. But then, when I fall on something I really wanna use fullscreen or even half-screen that is overly white, it burns both my and my girlfriend's retinas hahaha. So yeah it's not the best. Especially since I tend to be a nigh owl so I'm in the dark and it gets pretty intense at times how the backlash from going to everything very dark to a super bright white screen all of a sudden hahaha. 

So yeah if any of you would know if it'd be a possibility and also if it's something hard to do in general it'd be amazing please! <3

Edit: Can't believe I'm the first one to start a discussion on this GitHub hahaha.

**Comment by MalloyDelacroix** [comment]
I like the idea of a dark theme.  It is a lot different to implement such a thing on a desktop app than on a website.  It's definitely a non-trivial amount of work, as every UI file that we have would have to be at least somewhat changed.  I believe that the UI framework we use does allow for this, but I'm not overly familiar with how all of the styling settings work.

I will definitely look into this for future releases.  Style aspects are a lower priority on the issue list than bugs or hard features, but this is definitely a possibility.

**Reply by Proutbedaine** [reply]
Omg I wasn't notified of your reply and just saw it now while checking just in case! But thank you for replying and letting me know about it and give me an idea of how much work it might imply. I'm glad you will consider it for a future update someday! For me it's much more something practical than esthetics. As I mentioned it's much easier on my eyes. But I do get that of course debugging and other features are much more important still, in the end. Also beggars cannot be choosers hahaha, so I'll be patient obviously. Anyways rushing things is never a good thing ultimately. ^^
